v2.2/recommended-production-settings.md

```diff
@@ -39,54 +39,89 @@ Term | Definition
+Nodes should have sufficient CPU, RAM, network, and storage capacity to handle your workload. It's important to test and tune your hardware setup before deploying to production.
+
+#### Storage and memory recommendations
 
-- At a bare minimum, each node should have **2 GB of RAM and one entire core**. More data, complex workloads, higher concurrency, and faster performance require additional resources.
+- At a bare minimum, each node should have **2 GB of RAM and one entire core**. More data, complex workloads, higher concurrency, and faster performance require additional resources; as a general rule of thumb, increase the number of cores and do not oversubscribe memory to cores (maximum 8-16 GB of RAM per core).
     {{site.data.alerts.callout_danger}}
     Avoid "burstable" or "shared-core" virtual machines that limit the load on a single core.
     {{site.data.alerts.end}}
+- The recommended Linux filesystem is [ext4](https://ext4.wiki.kernel.org/index.php/Main_Page).
+- **[Need to add kernel settings]**
+- Avoid using shared storage.
+- Use [RAID 10](https://en.wikipedia.org/wiki/Nested_RAID_levels#RAID_10_(RAID_1+0)), which has minimal performance impact. RAID 0 and 1 are also acceptable from a performance perspective.
 
-- For best performance:
-    - Use SSDs over HDDs.
-    - Use larger/more powerful nodes. Adding more CPU is usually more beneficial than adding more RAM.
+#### Performance recommendations
 
-- For best resilience:
-    - Use many smaller nodes instead of fewer larger ones. Recovery from a failed node is faster when data is spread across more nodes.
-    - Use [zone configs](configure-replication-zones.html) to increase the replication factor from 3 (the default) to 5. This is especially recommended if you are using local disks rather than a cloud providers' network-attached disks that are often replicated underneath the covers, because local disks have a greater risk of failure. You can do this for the [entire cluster](configure-replication-zones.html#edit-the-default-replication-zone) or for specific [databases](configure-replication-zones.html#create-a-replication-zone-for-a-database), [tables](configure-replication-zones.html#create-a-replication-zone-for-a-table), or [rows](configure-replication-zones.html#create-a-replication-zone-for-a-table-or-secondary-index-partition) (enterprise-only).
-        {{site.data.alerts.callout_danger}}
-        {% include {{page.version.version}}/known-limitations/system-range-replication.md %}
-        {{site.data.alerts.end}}
+For best performance:
+
+- Use NVMe SSDs over HDDs.
+- To simulate a bare metal deployment where machines are directly connected to one physical disk each, create and mount a local SSD using a SCSI interface.
+- Use more powerful nodes. Adding more CPU is usually more beneficial than adding more RAM.
```

Comment:
I don't think this is true currently. We recommend 16 cpu nodes and adding more of them rather than using higher CPU nodes. We have some work in progress to address this for 2.2 but for now I'd say 16 cpu is the recommendation cc @nvanbenschoten

Reply:
But up to 16 cpus, adding more cpu is better than adding more ram. That's why we recommend `c` instances instead of `m` instances on aws. I'd recommend a range of 4-16 cpus per node.